cmd/contour/serve.go

```diff
@@ -677,7 +695,7 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 			log = log.WithField("insecure", true)
 		}
 
-		log.Info("started xDS server")
+		log.Infof("started xDS server type: %q version: %q", ctx.XDSServerType, xDSServerVersion)
```

Comment:
Maybe just add the server type and version as structured fields? e.g. 

```
log.WithField("xdsServerType", ctx.XDSServerType).WithField("xdsVersion", xDSServerVersion).Info("started xDS server")
```

Reply:
I modeled after the PR to refactor config out to the `pkg` package @skriss.

Reply:
I still prefer structured log fields but 🤷‍♂️